README.md

```diff
@@ -59,6 +59,21 @@ What options are available for each create and update request can be found in th
 ```
 ~/go/src/github.com/gridscale/gsclient-go
 ```
+## Examples
+Examples are in example folder:
```

Comment:
`examples` folder

Reply:
`An example of how to use resource can be found in examples folder:` would be better?